packages/contracts-bedrock/src/L1/SuperchainConfig.sol

```diff
@@ -71,39 +69,57 @@ contract SuperchainConfig is Initializable, ISemver {
     /// @notice Thrown when the input dependency is already added to the set.
     error DependencyAlreadyAdded();
 
+    /// @notice Thrown when a OptimismPortal does not have the right SuperchainConfig.
+    error InvalidSuperchainConfig();
+
     /// @notice Semantic version.
     /// @custom:semver 1.1.1-beta.5
     string public constant version = "1.1.1-beta.5";
 
+    /// @notice The Shared Lockbox contract
+    ISharedLockbox public sharedLockbox;
```

Comment:
I would recommend that we either use a spacer to move this away from being in the same slot as initialized or we use a custom setter/getter + a hash to define the slot

Reply:
slight preference for custom storage